packages/core/src/utils/connectors/index.ts

```diff
@@ -60,6 +61,20 @@ export const transpileConnectorFactory = ({
   return { type, ...metadata, isDemo: demoConnectorIds.includes(metadata.id) };
 };
 
+/**
+ * `extraInfo` is only used to expose email service vendors `fromEmail` setup to Logto email connector.
+ * Can extend this method in the future for other use cases.
+ */
+export const buildExtraInfoFromEmailServiceData = (
+  connectorFactoryId: string,
+  emailServiceProviderConfig?: EmailServiceData
+): ConnectorResponse['extraInfo'] => {
+  return conditional(
+    serviceConnectorIds.includes(connectorFactoryId) &&
```

Comment:
This `fromEmail` is for only one specific connector, can we use something like:
```ts
if (connectorFactroyId === logtoEmailConnectorId) {
```

Reply:
Updated.